src/test/java/com/google/javascript/clutz/provide_single_class.js

```diff
@@ -54,6 +54,6 @@ foo.bar.Baz.NestedClass = function() {};
 
 /** @enum */
 foo.bar.Baz.NestedEnum = {
-  A: 1,
+  XD: 1,
```

Comment:
what was the purpose of change to XD here? Maybe order? add a comment

Reply:
Initially for testing because I wasn't sure changes were being picked up, but then I decided to leave it to test order. (but I can revert this if I remove sorting given the message below)